content/applications/marketing/marketing_automation.rst

```diff
@@ -1,121 +1,191 @@
-becomes available on the campaign form.
+The contacts in the system that fit the specified criteria for a campaign are referred to as
+*records*.
 
-There is also the option to match records with :guilabel:`all` or :guilabel:`any` of the rules
-configured in the :guilabel:`Filter` field.
+The number of records that are displayed next to the campaign *Filter* represents the total number
+of records the campaign is targeting.
 
-To choose either of those options, click :guilabel:`all` from the middle of the sentence
-":guilabel:`Match records with all of the following rules`" to reveal a drop-down menu with those
-options.
+Participants
+------------
 
-.. image:: marketing_automation/match-all-any-rules-drop-down.png
-   :align: center
-   :alt: Match records with all or any of the rules in Filter field for marketing campaigns.
+The records that are engaged by the campaign are referred to as *participants*.
 
-When the first field of the rule equation is clicked, a nested drop-down menu of options appears on
-the screen where specific criteria is chosen based on needs of the campaign.
+The number of participants engaged in a test run are shown in the *Tests* smart button, which
+displays on the top of the campaign form after a test has been ran.
 
-The remaining fields on the rule equation further define the criteria, which is used to determine
-which records in the database to include or exclude in the execution of the campaign.
+The number of participants engaged in a running or stopped campaign are shown in the *Participants*
+smart button at the top of the campaign form.
 
-To add another rule, either click the :guilabel:`➕ (plus sign)` icon to the right of the filtering
-rule, or click :guilabel:`New Rule` beneath the rule equation fields. When either are clicked, a new
-series of rule fields appears.
+.. seealso::
+   :doc:`Targeting an audience <marketing_automation/target_audience>`
+
+Workflow
+========
+
+A *workflow* consists of an activity, many activities, or a sequence of activities organized in a
+campaign. A campaign's workflow is defined in the *Workflow* section of the campaign form.
+
+Activities
+----------
+
+*Activities* are the methods of communication or server actions, organized in a workflow, that are
+executed within a campaign. Once running, each activity displays the number of participants that
+are engaged by the activity as *Success* and *Rejected* counts.
+
+The following are the available activities in the *Marketing Automation* application:
+
+- *Email*: send an email to the target audience.
+- *SMS*: send an SMS to the target audience.
+- *Server Action*: executes an automated action.
+
+.. seealso::
+   :doc:`marketing_automation/workflow_activities`
+
+Testing and running
+===================
+
+Once a campaign has been created, it can be tested to ensure the workflow is functioning as expected
+and to check for errors and correct any mistakes before it reaches its target audience.
```

Comment:
```suggestion
Once a campaign has been created, it can be tested to ensure the workflow is functioning as
expected, to check for errors, and correct any mistakes before it reaches its target audience.
```